spec/factories/uploads.rb

```diff
@@ -1,4 +1,9 @@
 FactoryGirl.define do
   factory :upload do
+    content_type 'image/png'
+    association :user
+    after :create do |upload|
+      upload.update_column(:file, "upload.png")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.